cmd/updater/main.go

```diff
@@ -126,6 +126,7 @@ func gatherOptions() options {
 	return gatherFlagOptions(flag.CommandLine, os.Args[1:]...)
 }
 
+
```

Comment:
nit: I'm seeing a few formatting issues, `go fmt` should fix these (https://go.dev/blog/gofmt)

Reply:
The log for the failed test has an exact command to run.